src-tauri/src/main.rs

```diff
@@ -141,6 +144,9 @@ async fn get_processes(state: State<'_, AppState>) -> Result<(Vec<ProcessInfo>,
             memory_used: sys.used_memory(),
             memory_free: sys.total_memory() - sys.used_memory(),
             memory_cached: sys.total_memory() - (sys.used_memory() + (sys.total_memory() - sys.used_memory())),
+            swap_space: sys.total_swap(),
+            swap_used: sys.used_swap(),
+            swap_free: sys.free_swap(),
```

Comment:
where are the svelte template changes ? do you prefer if I work on those by myself?

Reply:
Yeah i would prefer that